### PR TITLE
Convert manager AMD modules to ES6

### PR DESCRIPTION
**Pull request by danielnaab**

### CODE REVIEW

File: assets/package.json [hunk 1]
```diff
@@ -73,6 +76,9 @@
     "uglifycss": "0.0.29"
   },
   "dependencies": {
+    "font-awesome": "^4.6.3",
+    "jquery": "^2.2.4",
+    "jquery-ui": "^1.12.0",
     "less": "2.7.1"
```
Comment (mbucknell) [inline assets/package.json]: Should less by a devDependency?

File: assets/scripts/manager/models/PublicationContributorModel.js [hunk 2]
```diff
@@ -1,41 +1,37 @@
...
-            if (!_.has(response, 'affiliation')) {
-                response.affiliation = affiliation;
+    fetch : function(options) {
+        var params = {
+            data : {
+                mimetype : 'json'
             }
-            return response;
+        };
+        if (_.isObject(options)) {
```
Comment (mbucknell) [inline assets/scripts/manager/models/PublicationContributorModel.js]: Need an import for underscore. This is also flagged by Codacy.

File: assets/tests/js/manager/test-manifest.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+require('./test-config');
+
+
+// This loads all the tests in the manager bundle.
+require('./models/ContributorModelSpec');
```
Comment (mbucknell) [inline assets/tests/js/manager/test-manifest.js]: I can see this becoming a problem where tests are written but not actually run because they weren't added to the manifest (I know we have the same issue in WQP)

File: assets/scripts/manager/vendor-globals.js [hunk 4]
```diff
@@ -0,0 +1 @@
+window.$ = window.jQuery = require('jquery');
```
Comment (mbucknell) [inline assets/scripts/manager/vendor-globals.js]: Why is this necessary? It seems like we are importing this anyways where it is needed?

File: assets/tests/js/manager/test-manifest.js (hunk 3 above)
Comment (danielnaab) [reply assets/tests/js/manager/test-manifest.js]: Yes, I agree. This was a hack I stumbled onto when updating WDFN to make "npm run test:watch" tasks run at a reasonable speed. Here's some more info on the problem: https://github.com/jlmakes/karma-rollup-preprocessor/issues/28

Webpack has a similar problem. I'd love a better solution...

File: assets/scripts/manager/vendor-globals.js (hunk 4 above)
Comment (danielnaab) [reply assets/scripts/manager/vendor-globals.js]: Backbone expects jQuery to be a global when it's imported. I'll see if I can bake this part into the Rollup config in some manner, once I get to switching all the bower packages to npm.
